.github/workflows/build_and_test.yaml

```diff
@@ -42,6 +39,10 @@ jobs:
           name: codecov-envoy-gateway
           verbose: true
 
+      # build
+      - run: make build-multiarch
+        if: github.event_name == 'pull_request'
```

Comment:
nice, even though there is a build cache, this should save us some setup time for the push case

Reply:
Yes, we should keep our workflows as clean as possible, do not run repeated and unuseful jobs.